cypress/support/util/keycloak_hooks.ts

```diff
@@ -1,15 +1,25 @@
+import AdminClient from "./AdminClient";
+
+export const TEST_REALM = "test" as const;
```

Comment:
Constant variables that are assigned a string literal are already a constant type by default.
```suggestion
export const TEST_REALM = "test";
```